### PR TITLE
feat(suite): Account navigation updates + Account sticky menu

### LINKED ISSUE #3348: Account navigation
New account navigation design improvements: 

- [x] Increase font size for the fiat value (from 20 to 24px) [Zeplin](https://app.zeplin.io/project/5ee87c00f6af719a645d14c3/screen/601d23fb81d8f9849d65adb8)
- [x] Tabs for `Overview` and  `Details` (i.e. remove 3 dots account menu) 
- [x] Turn tabs into secondary buttons for `Send`, `Receive`, `Trade` 
- [x] `‹ Back` button for expanded Send, Receive, Trade views [Zeplin](https://app.zeplin.io/project/5ee87c00f6af719a645d14c3/screen/601d23f300e5519f7f8c7da2) 

- [x] Sticky account header [Zeplin](https://app.zeplin.io/project/5ee87c00f6af719a645d14c3/screen/601d2d464c342e10b482dc94)

Please disregard other design changes (i.e. navbar, left account menu, top navigation etc.) in the proposal for now. These are still being tested and reviewed. This issue is related to the account header only.

Screen Shot 2021-02-05 at 12 05 40

### PR DESCRIPTION
**Pull request by tomexx**
implements #3348

### CODE REVIEW

File: packages/suite/src/components/suite/AppNavigation/index.tsx [hunk 1]
```diff
@@ -1,19 +1,23 @@
-import React from 'react';
+import React, { useRef, useLayoutEffect, useState } from 'react';
 import styled from 'styled-components';
-import { Icon, variables, IconProps, useTheme } from '@trezor/components';
+import { variables, IconProps, useTheme, Button, Icon, Dropdown } from '@trezor/components';
 import { Translation } from '@suite-components';
 import { useSelector, useActions } from '@suite-hooks';
 import * as routerActions from '@suite-actions/routerActions';
 import { Route } from '@suite-types';
+import useResizeObserver from '@react-hook/resize-observer';
```
Comment (szymonlesisz) [inline packages/suite/src/components/suite/AppNavigation/index.tsx]: we already have debounced resize handler in [resizeActions](https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/actions/suite/resizeActions.ts) and state stored in [resizeReducer](https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/reducers/suite/resizeReducer.ts)
do we need another dependency for that?

File: packages/suite/src/components/suite/AppNavigation/index.tsx [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
 const { FONT_WEIGHT, FONT_SIZE } = variables;
 
+const SECONDARY_MENU_BUTTON_MARGIN = '12px';
+
 const Wrapper = styled.div`
     display: flex;
     width: 100%;
-    border-top: 1px solid ${props => props.theme.STROKE_GREY};
-    overflow-x: auto;
+    /* overflow-x: auto; */
```
Comment (szymonlesisz) [inline packages/suite/src/components/suite/AppNavigation/index.tsx]: rm leftover comment

**Review by szymonlesisz — CHANGES_REQUESTED** [review]

File: packages/suite/src/components/suite/AppNavigation/index.tsx (hunk 1 above)
Comment (tomexx) [reply packages/suite/src/components/suite/AppNavigation/index.tsx]: I used that, thanks

**Comment by tsusanka** [comment]
@tomexx let's finish this :rocket:. Also please note the double "Changed" in your changelog. https://github.com/trezor/trezor-suite/pull/3485/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR9-R14

File: packages/suite/src/components/wallet/AccountTopPanel/components/AccountNavigation/index.tsx [hunk 3]
```diff
@@ -1,30 +1,106 @@
...
+            isHidden: () => false,
+        },
+        {
+            id: 'wallet-coinmarket-buy',
+            callback: () => {
+                goto('wallet-coinmarket-buy', undefined, true);
+            },
+            title: <Translation id="TR_NAV_TRADE" />,
+            position: 'secondary',
+            isHidden: () => false,
```
Comment (szymonlesisz) [inline packages/suite/src/components/wallet/AccountTopPanel/components/AccountNavigation/index.tsx:66]: hm, idk why but this button is rendered as primary:
Screenshot from 2021-04-06 09-24-08

Comment (szymonlesisz) [reply packages/suite/src/components/wallet/AccountTopPanel/components/AccountNavigation/index.tsx:66]: zeplin seems to be outdated, resolving

**Review by szymonlesisz — APPROVED** [review]

**Pull request merged**